src/poliastro/twobody/orbit.py

```diff
@@ -1497,3 +1504,294 @@ def plot(self, label=None, use_3d=False, interactive=False):
             from poliastro.plotting.core import OrbitPlotter2D
 
             return OrbitPlotter2D().plot(self, label=label)
+
+    @classmethod
+    def __groundtrack_orbit(
```

Comment:
Let's use a single underscore here, to avoid [name mangling](https://www.python.org/dev/peps/pep-0008/#designing-for-inheritance) (we just want to make the method "private")